src/app/external-cluster-wizard/steps/external-cluster/provider/eks/component.ts

```diff
@@ -97,13 +124,21 @@ export class EKSClusterSettingsComponent
       [Controls.Vpc]: this._builder.control(null, Validators.required),
       [Controls.SubnetIds]: this._builder.control([], Validators.required),
       [Controls.SecurityGroupsIds]: this._builder.control([], Validators.required),
+      [Controls.DiskSize]: this._builder.control('20'),
+      [Controls.MaxSize]: this._builder.control('1'),
+      [Controls.MinSize]: this._builder.control('1'),
+      [Controls.DesiredSize]: this._builder.control('1'),
     });
   }
 
   private _initSubscriptions(): void {
     this.form.valueChanges.pipe(takeUntil(this._unsubscribe)).subscribe(_ => {
       this._updateExternalClusterModel();
+      this._getExternalMachineDeployment();
+      this.maxNodeCount = this.controlValue(Controls.MaxSize);
+      this.minNodeCount = this.controlValue(Controls.MinSize);
       this._externalClusterService.isClusterDetailsStepValid = this.form.valid;
+      this._externalMachineDeploymentService.isAddMachineDeploymentFormValid = this.form.valid;
```

Comment:
As per our discussion. 

- Currently wizard has all fields which are needed so form becomes valid because controls which are not shown in wizard e.g DiskSize, DesiredSize has default values so overall form validity.

- Validators are set/removed dynamically based on Cluster Create or MD create therefore works as expected.

**Resolving this issues after clarification:.**